test/MIEngine.regdef

```diff
@@ -7,6 +7,7 @@ Windows Registry Editor Version 5.00
 ; "AddressBP"=dword:00000001
 "AlwaysLoadLocal"=dword:00000001
 "AlwaysLoadProgramProviderLocal"=dword:00000001
+"BpBindingLongBindExtension"=dword:00000001
```

Comment:
Might want to add a comment that this isn't in the normal registration and what it is for.